src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx

```diff
@@ -0,0 +1,52 @@
+import { TextField } from '@mui/material';
+import styles from '../../../HousingLottery.module.css';
+
+const ApplicantFields = ({ applicant, onApplicantChange, index }) => {
+  const handleFieldChange = (field, value) => {
+    onApplicantChange(index, { ...applicant, [field]: value });
+  };
+
+  return (
+    <>
+      <TextField
+        type="text"
+        variant='outlined'
+        color='secondary'
+        label="First Name"
+        onChange={(e) => handleFieldChange('firstName', e.target.value)}
+        value={applicant.firstName}
+        fullWidth
+        required
+        className={styles.applicant_name}
+        helperText={'*Required'}
+      />
+      <TextField
+        type="text"
+        variant='outlined'
+        color='secondary'
+        label="Last Name"
+        onChange={(e) => handleFieldChange('lastName', e.target.value)}
+        value={applicant.lastName}
+        fullWidth
+        required
+        className={styles.applicant_name}
+        helperText={'*Required'}
+      />
+      <TextField
+        type="email"
+        variant='outlined'
+        color='secondary'
+        label="Email"
+        onChange={(e) => handleFieldChange('email', e.target.value)}
+        value={applicant.email}
+        required
+        error={!applicant.email.endsWith('@gordon.edu')}
+        helperText={!applicant.email.endsWith('@gordon.edu') ? 'Not a Valid Gordon Email' : '*Required'}
```

Comment:
Can you add a check for empty string, so when there is no input, the text field would not be red like now?
![image](https://github.com/gordon-cs/gordon-360-ui/assets/78650688/317dfaac-e8f9-4c47-b81f-7c0ce182713e)